appcelerator/titanium_mobile
# fix(android): changing picker row color should update selected text

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-27927

**Test:**
1. Build and run the below on Android.
2. Tap on the picker to show the drop-down box.
3. Tap on "Green".
4. Verify that the "Green" text has changed to purple. _(This is the fix.)_
5. Tap on the picker to show the drop-down box.
6. Verify the "Green" text is purple in the drop-down.

```javascript
var window = Ti.UI.createWindow({ backgroundColor: "gray" });
var picker = Ti.UI.createPicker({ width: "50%" });
picker.add([
	Ti.UI.createPickerRow({ title: "Red", color: "red" }),
	Ti.UI.createPickerRow({ title: "Green", color: "green" }),
	Ti.UI.createPickerRow({ title: "Blue", color: "blue" }),
]);
picker.addEventListener("change", function(e) {
	Ti.API.info("### Selected row: " + e.rowIndex);
	picker.getSelectedRow(0).color = "purple";
});
window.add(picker);
window.open();
```
